target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -206,7 +128,6 @@ ramips_setup_macs()
 	dlink,dir-860l-b1)
 		lan_mac=$(mtd_get_mac_ascii factory lanmac)
 		wan_mac=$(mtd_get_mac_ascii factory wanmac)
-		label_mac=$(mtd_get_mac_binary radio 0x4)
```

Comment:
That's still required, unless you can read the MAC address from gmac0 in device tree.

Reply:
> unless you can read the MAC address from gmac0 in device tree.

You can, gmac0 is eth0